### DISCUSSION TITLE
Get the full ubuntu desktop experience via xRDP

### DISCUSSION THREAD

**Discussion by mmrad1** [neutrinolabs/xrdp#1915] [Q&A] [answered]
hi I am referring to this post [here:](https://superuser.com/questions/1479313/how-do-i-get-the-full-ubuntu-gnome-experience-via-xrdp) . The question here is
"
How do I get the full Ubuntu Gnome experience instead of the generic Gnome experience via XRDP?

When I login from the console, I get the Ubuntu customizations of Gnome: The dock is a stripe that extends from the one edge of the screen to the other, and the Settings app has a Dock entry.

When I login via XRDP, it looks like Generic Gnome: The dock is only large enough to contain its icons, and Settings has no Dock entry. More importantly, I have a bunch of icons on my "Ubuntu" desktop which are absent from the "generic" desktop...."

Is there any option here? Maybe this thread is a duplicate and my apologies if it is but I did not find any related threads

**Comment by matt335672** [comment, marked as answer]
Hi @andymcraae1 

Is this the same as #1723?

**Reply by mmrad1** [reply]
thanks a lot.

**Reply by matt335672** [reply]
Did this help @andymcraae1 ?

We're a bit new to the discussions feature here, but if the above was useful we can makr it as an 'answer'.

Thanks

**Reply by mmrad1** [reply]
Yep. Its super cool. I just posted also the solution of this issue on askubuntu https://superuser.com/questions/1479313/how-do-i-get-the-full-ubuntu-gnome-experience-via-xrdp/1655262#1655262. Thanks a lot
